core/src/toga/widgets/imageview.py

```diff
@@ -124,3 +133,23 @@ def image(self, image):
 
         self._impl.set_image(self._image)
         self.refresh()
+    
+    def as_image(self, format: Any | None=None):
+        '''
+        get the image from ImageView as specified format if supported
+        :param format: None or A supported type of Image
+        Supported types are `PIL.Image.Image`,
+        :returns: toga.Image if format is None, or the specified format if the format is supported
+        ```
+        from PIL import Image
+        pil_image = imageview.as_image(Image.Image)
+        ```
+        '''
+        if format==None:
+            return self.image
+        elif isinstance(format, Image):
+            return self.image
```

Comment:
If two branches have the same implementation, perhaps consider whether an `or` is needed.